js/controller.js

```diff
@@ -1293,10 +1293,16 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
       // no support
       else {
-        if(this.state.isFullWindow) {
-          this.exitFullWindow();
+
+        if(OO.isIos && this.videoVr){
+          // toggle fullscreen with bitmovin api
+          this.mb.publish(OO.EVENTS.TOGGLE_FULLSCREEN_VR, this.focusedElement);
```

Comment:
Can we add more details about why the usual fullscreen mechanism doesn't work for VR?

Reply:
this.state.isVideoFullScreenSupported == false;
this.state.isFullScreenSupported == false; 
The rest also did not work. fullscreen bitmovin works correctly

Reply:
Can you add this in as a comment?